applications/GeoMechanicsApplication/custom_processes/apply_boundary_hydrostatic_pressure_table_process.hpp

```diff
@@ -48,40 +44,25 @@ class ApplyBoundaryHydrostaticPressureTableProcess : public ApplyConstantBoundar
         KRATOS_CATCH("")
     }
 
-    ///------------------------------------------------------------------------------------
-
-    /// Destructor
-    ~ApplyBoundaryHydrostaticPressureTableProcess() override {}
-
-///----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
-
-    /// Execute method is used to execute the ApplyBoundaryHydrostaticPressureTableProcess algorithms.
-    void Execute() override
-    {
-    }
+    ApplyBoundaryHydrostaticPressureTableProcess(ApplyBoundaryHydrostaticPressureTableProcess const& rOther) = delete;
+    ApplyBoundaryHydrostaticPressureTableProcess& operator=(ApplyBoundaryHydrostaticPressureTableProcess const& rOther) = delete;
```

Comment:
I would like to propose two minor stylistic changes:
1. Let `const` precede the type name rather than follow it.
2. Since there is no implementation (they are `delete`d), there is no need to name the function parameters.

```suggestion
    ApplyBoundaryHydrostaticPressureTableProcess(const ApplyBoundaryHydrostaticPressureTableProcess&) = delete;
    ApplyBoundaryHydrostaticPressureTableProcess& operator=(const ApplyBoundaryHydrostaticPressureTableProcess&) = delete;
```

Reply:
Done in all places I could find in custom_processes.